### PR TITLE
improve troubleshooting

### PR DESCRIPTION
**Pull request by Gruetzig**
saves space and adds more troubleshooting

### CODE REVIEW

File: _pages/en_US/troubleshooting.txt [hunk 1]
```diff
@@ -146,21 +142,33 @@ If neither of these solutions fixes this problem, then custom firmware may have
 
 There may be an issue with your `F00D43D5.bin` file (it may be corrupted or intended for the wrong 3DS). Re-create your `F00D43D5.bin` file with the [Bannerbomb3 Injector](http://3dstools.nhnarwhal.com/#/bb3gen), ensuring that you use the `movable.sed` file for your console.
 
-### DSiWare Management menu crashes without purple screen
+---
+
+## unSAFE_MODE
```
Comment (lifehackerhansol) [inline _pages/en_US/troubleshooting.txt:147]:
```suggestion
## Installing boot9strap (USM)
```

File: _pages/en_US/troubleshooting.txt [hunk 2]
```diff
@@ -146,21 +142,33 @@ If neither of these solutions fixes this problem, then custom firmware may have
...
+
+### Failed to mount the SD card!
+
+Back up your data and reformat your SD card as FAT32 with the recommended tool depending on your operating system ([Windows](formatting-sd-(windows)), [macOS](formatting-sd-(mac)), [Linux](formatting-sd-(linux))). MiniTool Partition Wizard and the HP formatting tool (HPUSBDisk) are known to cause issues with 3DS SD cards.
+
+If this is unsuccessful, try using another SD card.
+
+---
+
+## Fredtool
```
Comment (lifehackerhansol) [inline _pages/en_US/troubleshooting.txt:171]:
```suggestion
## Installing boot9strap (Fredtool)
```

File: _pages/en_US/troubleshooting.txt [hunk 3]
```diff
@@ -124,19 +124,15 @@ If this is unsuccessful, try using another SD card.
 
 ---
 
-## Installing boot9strap (USM)
+## BannerBomb3
 
-### DSiWare Management menu crashes without showing BB3 multihax menu (USM)
+### DSiWare Management menu crashes without showing BB3 multihax menu (unSAFE_MODE and Fredtool)
```
Comment (lifehackerhansol) [inline _pages/en_US/troubleshooting.txt:129]:
```suggestion
### DSiWare Management menu crashes without showing BB3 multihax menu
```
If we're merging all of them, do we need to specify the exploit?

**Review by lifehackerhansol — CHANGES_REQUESTED** [review]
I'm also generally not sure if I like merging the two. I feel there's always going to be some confusion about the SAFE_MODE true/false statement, which is why I didn't merge them.

File: _pages/en_US/troubleshooting.txt (hunk 3 above)
Comment (Gruetzig) [reply _pages/en_US/troubleshooting.txt:129]: well i thought this works to distinguish the 2, fredtool and unsafe mode, and Bannerbomb+Fredtool (TWN), but we don't need to

**Comment by Gruetzig** [comment]
i guess I revert 80% of my thoughts by commiting the requested changes

**Comment by lifehackerhansol** [comment]
Well I mean the rest of it is okay... need to triple check lol

**Comment by lifehackerhansol** [comment]
Last thing before I merge this, because it looks okay. Can you add a FAQ for fredtool where one may not have deleted `F00D43D5.bin` and it's still showing bb3 multihax when trying to cpy flipnote over?

**Comment by lifehackerhansol** [comment]
Oh also now that the troubleshooting names changed we gotta fix the inline links to them in the respective pages

**Comment by Gruetzig** [comment]
I hope I understand "FAQ" correctly here

**Comment by lifehackerhansol** [comment]
Sorry, wrong choice of words. I keep mixing the two.

**Comment by Gruetzig** [comment]
It should be only 2?

**Review by lifehackerhansol — APPROVED** [review]

**Pull request merged**